packages/main/src/GridRow.ts

```diff
@@ -85,7 +77,6 @@ class GridRow extends UI5Element {
 	_invalidate!: number;
 
 	static i18nBundle: I18nBundle;
-
 	static async onDefine() {
 		GridRow.i18nBundle = await getI18nBundle("@ui5/webcomponents");
```

Comment:
Don't think we need to assign the i18nBundle here. Could just call GridRowBase.onDefine() here, no?